test/stage1/behavior/align.zig

```diff
@@ -220,6 +220,30 @@ test "alignment of structs" {
     }) == @alignOf(usize));
 }
 
+test "alignment of >= 128-bit integer type" {
+    expect(@alignOf(u128) == 16);
+    expect(@alignOf(u129) == 16);
+}
+
+test "alignment of struct with 128-bit field" {
+    expect(@alignOf(struct {
+        x: u128,
+    }) == 16);
+
+    comptime {
+        expect(@alignOf(struct {
+            x: u128,
+        }) == 16);
+    }
+}
+
+test "size of struct with non-standard-LLVM alignment" {
+    expect(@sizeOf(struct {
```

Comment:
zig doesn't guarantee the size of non-extern, non-packed structs. You can fix this test case by making it `extern struct`.

Reply:
I replaced this specific test with the other one you suggested above, since 129-bit integers are not allowed in `extern structs` which made this specific one obsolete.